### PR TITLE
[FE] Social media bar

### PR DESCRIPTION
**Pull request by TungDude**
## Issue ID/Name
https://linear.app/oddsnends/issue/ODD-16/[fe]-social-media-bar

## Description
Add a social media bar into the web layout, and test the page content’s overflow to ensure the bar stays visible on the screen, even with additional content.

## Checklist
- [X] I have self-reviewed my code.
- [X] I have commented on my code, particularly in the hard-to-understand area.
- [X] (Should Be) I have NOT committed the modified environment/config file(s) to this PR.
- [X] I have tagged a reviewer for this PR.

## Screenshot(s)
Screenshot 2568-01-26 at 17 29 31

## Reviewer
@LittleNa1000 

## Remark
I’d like your opinion on whether we should have the social media links open in a new tab or the same tab. Personally, I think it should open in a new tab so users can still keep our page open, but idk.

### CODE REVIEW

**Comment by LittleNa1000** [comment]
Deploy preview for _oddsnends_ ready!

✅ Preview
https://oddsnends-iqsezrtnv-oddsnends-projects.vercel.app

Built with commit 7e7433f64ea50ebacc27085e16c3d4acc2e73156.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

File: src/app/layout.tsx [hunk 1]
```diff
@@ -16,8 +18,12 @@ export default function RootLayout({
   return (
     <html lang="en" className={`${GeistSans.variable}`}>
       <LiffProvider>
-        <body className="relative mx-auto min-h-screen w-full max-w-md overflow-x-hidden overscroll-none">
-          {children}
+        <body className={`flex flex-col h-screen`}>
```
Comment (LittleNa1000) [inline src/app/layout.tsx:21]: please keep the root layout className as original for the mobile display

**Review by LittleNa1000 — CHANGES_REQUESTED** [review]

File: src/app/layout.tsx (hunk 1 above)
Comment (TungDude) [reply src/app/layout.tsx]: Done krub. Could you please review it again?

**Review by LittleNa1000 — APPROVED** [review]
lgtm
